SCClassLibrary/Common/Quarks/QuarksGui.sc

```diff
@@ -328,7 +328,7 @@ QuarkDetailView {
 
 			model.data.keysValuesDo({ |k, v|
 				if(#[\name, \summary, \url, \path, \dependencies, \version].includes(k).not) {
-					this.pushRow(k.asString, v.asString);
+					this.pushRow(k.asString, v.value().asString);
```

Comment:
I think it needs to protect this if its going to value it. 

but it should do it I guess.